### PR TITLE
Return "check-digit" field in "hbclaim" table from API

### PR DESCRIPTION
**Pull request by elcorbs**
We will need this field for the integration with single view. 

- Added the `hbclaim` table to the DMS selection rules
- returned the field from both endpoints

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -1,210 +1,424 @@
...
-	rented_prop smallint NULL,
-	own_telno varchar(20) NULL,
-	own_email_addr varchar(128) NULL,
-	no_beds smallint NULL,
-	annexe_ind smallint NULL,
-	main_prop_ref varchar(18) NULL,
-	own_is_charity smallint NULL,
-	own_addr_abroad smallint NULL
-);
+CREATE TABLE hbhousehold (
```
Comment (elcorbs) [inline database/schema.sql:1]: I'm not sure why the git diff thinks I've changed the whole file. But the changes I actually made were the addition of the table on line 104.

**Review by LBHBomotoso — APPROVED** [review]
Looks good

File: AcademyResidentInformationApi.Tests/V1/Gateways/AcademyGatewayTests.cs [hunk 2]
```diff
@@ -233,10 +238,7 @@ public void GetAllResidentsWithNameAndPostCodeQueryParameterReturnsMatchingResid
         public void GetAllResidentsWithPostCodeQueryParameterIgnoresFormatting(string postcode)
         {
             var databaseEntity = AddPersonRecordToDatabase();
-
-            var address = TestHelper.CreateDatabaseAddressForPersonId(databaseEntity.ClaimId, databaseEntity.HouseId, postcode);
-            AcademyContext.Addresses.Add(address);
-            AcademyContext.SaveChanges();
+            var address = AddAddressToDatabase(databaseEntity.ClaimId, databaseEntity.HouseId, postcode: postcode);
```
Comment (louisebolden) [inline AcademyResidentInformationApi.Tests/V1/Gateways/AcademyGatewayTests.cs:241]: The `postcode: postcode` argument here is an argument with a label? Is [this](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments#named-arguments) the right bit of the docs for me to read up?

Comment (elcorbs) [reply AcademyResidentInformationApi.Tests/V1/Gateways/AcademyGatewayTests.cs:241]: Yep and yes to the docs as well :)

**Review by louisebolden — APPROVED** [review]
Good steps towards Single View integration 👍

**Review by makeyw — APPROVED** [review]
Looks good - like the Named arguments bit too...didn't know that :)

**Pull request merged**